src/java/fr/paris/lutece/portal/business/portlet/AliasPortletHome.java

```diff
@@ -82,7 +82,10 @@ public static PortletHome getInstance( )
     {
         if ( _singleton == null )
         {
-            _singleton = new AliasPortletHome( );
+            synchronized ( AliasPortletHome.class )
+            {
+                _singleton = new AliasPortletHome( );
+            }

```

Comment:
Several threads can see _singleton as `null` and then instantiate each a new `AliasPortletHome`. You should just initialize _singleton at the point of declaration. It will be guarented to be unique by the way Java instantiate classes.

Reply:
Corrected